src/components/video/VideoPlayerHls.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+    <div class="w-full h-full relative">
+        <video class="w-full h-full rounded-md object-cover focus:outline-gray-500"
```

Comment:
FYI: it may look better to move class to the new line and do like that everywhere, but that are my preferences